pkg/base/common.lua

```diff
@@ -177,6 +177,9 @@ TOOL_GUN = 2
 TOOL_EXPL = 3
 TOOL_MARKER = 4
 
+--respawn
+cvar("MODE_SPECTATE", true)
```

Comment:
This would be better a few lines up, before the tools section, so it's with the other cvars.